flocker/node/test/test_deploy.py

```diff
@@ -619,7 +619,9 @@ def test_does_not_exist(self):
     name=APP_NAME2,
     image=DockerImage.from_string(UNIT_FOR_APP2.container_image)
 )
-EMPTY_NODESTATE = NodeState(hostname=u"example.com")
+EMPTY_NODESTATE = NodeState(hostname=u"example.com",
+                            manifestations={}, applications=[],
+                            used_ports=[], devices={}, paths={})
```

Comment:
Unfortunately until we remove the hack where UUID is calculated from hostname I don't want to unify stuff too much, since probably different pieces of code have different expectations.